sanitycheck prints some build errors directly to the console

```
$ sanitycheck -t userspace 
INFO    - JOBS: 32
INFO    - Selecting default platforms per test case
INFO    - Building initial testcase list...
INFO    - 748 test configurations selected, 156511 configurations discarded due to filters.
INFO    - Adding tasks to the queue...
INFO    - Total complete:   17/ 748   2%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

ERROR   - WARNING: Using a test - not safe - entropy source

ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:   21/ 748   2%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:   22/ 748   2%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:   23/ 748   3%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:   24/ 748   3%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:   26/ 748   3%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:   34/ 748   4%  skipped:   16, failed:    0ERROR   - WARNING: Using a test - not safe - entropy source

INFO    - Total complete:  228/ 748  30%  skipped:   48, failed:    0ERROR   - make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Terminated
make[2]: *** [CMakeFiles/Makefile2:478: zephyr/CMakeFiles/run.dir/all] Terminated
make[1]: *** [CMakeFiles/Makefile2:485: zephyr/CMakeFiles/run.dir/rule] Terminated
make: *** [Makefile:222: run] Terminated

INFO    - Total complete:  246/ 748  32%  skipped:   48, failed:    0ERROR   - /bin/sh: 1: MDB-NOTFOUND: not found
make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Error 127
make[2]: *** [CMakeFiles/Makefile2:478: zephyr/CMakeFiles/run.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:485: zephyr/CMakeFiles/run.dir/rule] Error 2
make: *** [Makefile:222: run] Error 2

ERROR   - make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Terminated
                                                                                                  make[2]: *** [CMakeFiles/Makefile2:511: zephyr/CMakeFiles/run.dir/all] Terminated
                                                                           make[1]: *** [CMakeFiles/Makefile2:518: zephyr/CMakeFiles/run.dir/rule] Terminated
                                                     make: *** [Makefile:235: run] Terminated

INFO    - Total complete:  249/ 748  33%  skipped:   48, failed:    0ERROR   - make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Terminated
                                                               make[2]: *** [CMakeFiles/Makefile2:1207: zephyr/CMakeFiles/run.dir/all] Terminated
                                         make[1]: *** [CMakeFiles/Makefile2:1214: zephyr/CMakeFiles/run.dir/rule] Terminated
                    make: *** [Makefile:469: run] Terminated

                                                            ERROR   - make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Terminated
make[2]: *** [CMakeFiles/Makefile2:1201: zephyr/CMakeFiles/run.dir/all] Terminated
make[1]: *** [CMakeFiles/Makefile2:1208: zephyr/CMakeFiles/run.dir/rule] Terminated
make: *** [Makefile:469: run] Terminated

ERROR   - make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Terminated
                                                                                                  make[2]: *** [CMakeFiles/Makefile2:1207: zephyr/CMakeFiles/run.dir/all] Terminated
                                                                            make[1]: *** [CMakeFiles/Makefile2:1214: zephyr/CMakeFiles/run.dir/rule] Terminated
                                                       make: *** [Makefile:469: run] Terminated

                                                                                               ERROR   - make[3]: *** [zephyr/CMakeFiles/run.dir/build.make:57: zephyr/CMakeFiles/run] Terminated
make[2]: *** [CMakeFiles/Makefile2:1201: zephyr/CMakeFiles/run.dir/all] Terminated
make[1]: *** [CMakeFiles/Makefile2:1208: zephyr/CMakeFiles/run.dir/rule] Terminated
make: *** [Makefile:469: run] Terminated

INFO    - Total complete:  257/ 748  34%  skipped:   48, failed:    0

```

Looks like in some cases, build stderr is going out the console instead of being logged somwhere

--- a/scripts/sanitycheck
+++ b/scripts/sanitycheck
@@ -576,9 +576,6 @@ class BinaryHandler(Handler):
                 t.join()
             proc.wait()
             self.returncode = proc.returncode
-            _, stderr = proc.communicate(timeout=30)
-            if stderr:
-                logger.error(stderr.decode())
 
         handler_time = time.time() - start_time
 

